### DISCUSSION TITLE
[Q] AsyncTCP_SSL ?

### DISCUSSION THREAD

**Discussion by brodrigz** [mathieucarbou/AsyncTCP#22] [Q&A] [answered]
Are there any async https implementations using this lib?, the original AsyncHTTPSRequest_Generic is extremely buggy and unstable, looking at the closed issues i can see many of the problems I'm facing were fixed on this fork

**Comment by mathieucarbou** [comment, marked as answer]
Hello,
There is no ssl support (and won't be except if someone contributes).
[AsyncTCPSock](https://github.com/yubox-node-org/AsyncTCPSock), which is somehow API-compatible, supports ssl.
